pmd-xml/src/main/resources/category/pom/errorprone.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+
+<ruleset name="Errorprone"
+         xmlns="http://pmd.sourceforge.net/ruleset/2.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/2.0.0 http://pmd.sourceforge.net/ruleset_2_0_0.xsd">
+
+    <description>
+Rules to detect constructs that are either broken, extremely confusing or prone to runtime errors.
+    </description>
+
+    <rule name="InvalidDependencyTypes"
+          language="pom"
+          since="5.4"
+          message="By default, Maven only recognized the following types: $validTypes."
```

Comment:
only recognizes

Also, does this interpolation work? the property definition is commented below...

Reply:
Nope, this interpolation doesn't work, because the property is not defined :) I'll replace the placeholder for now directly.